### PR TITLE
Working with CSV files in Python

### LINKED ISSUE #4773: [Languages] Working with CSV files in Python
#### Proposed title of article
[Languages] Working with csv files in Python

#### Proposed article introduction
CSV (Comma Separated Values) is a straightforward file format for storing tabular data in spreadsheets and databases. A CSV file is a plain text file that contains tabular data (numbers and text). A data record is represented by each line in the file. Each record has one or more fields, which are separated by commas. The name for this file format comes from the usage of the comma as a field separator.

#### Key takeaways 
By the end of this tutorial, the reader should be able to understand:
- what CSV is
- how to read a CSV file
- how to write a CSV file
- how to write a dictionary to a CSV file
- how to work with large CSV files in Python
- how to convert multiple JSON files to CSV Python
- Creating a dataframe using CSV files
- Conclusion

#### Article quality
By the end of this article, the reader will have acquired the needed knowledge on CSV  and the working of CSV files.

#### References
N/A

### PR DESCRIPTION
**Pull request by carteblanchemunene**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This fixes #4773

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Please always use the template provided by us.
I edited your initial comment and added the template. Check what matches your article.

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

But I think this article is light on substance - What do you think? @paulodhiambo

File: content/articles/working-with-csv-files-in-python/index.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+df.sample(10) # This specifies the time taken, 10 seconds
+
+```
+
+![Output](/engineering-education/working-with-csv-files-in-python/image1.png)
+
+
+### Convert multiple JSON files to CSV files
+A JSON file contains basic data structures and objects in JavaScript Object Notation (JSON). Sending data between an internet app and a server is the most common use case.
```
Comment (paulodhiambo) [inline content/articles/working-with-csv-files-in-python/index.md:143]: Change "Sending data between an internet app and a server is the most common use case." to "The most common use case is sending data between an internet app and a server."

**Comment by paulodhiambo** [comment]
@carteblanchemunene add more content to the article as @ahmadmardeni1 said, I suggest you add more content on how to manipulate the data after reading from the .csv files. I'll close this PR for now to clear the queue. Once you are done, please ping me to reopen this PR.

**Comment by carteblanchemunene** [comment]
Hello @paulodhiambo Please reopen the article for me so that I can commit the changes.

**Comment by carteblanchemunene** [comment]
@paulodhiambo please reopen this for me

**Comment by carteblanchemunene** [comment]
Hello @paulodhiambo please check my last commit

**Comment by carteblanchemunene** [comment]
Hello @paulodhiambo I have done the requested changes.

**Comment by carteblanchemunene** [comment]
Hello @paulodhiambo is this not a tech tutorial because you have not added the symbol!

**Comment by hectorkambow** [comment]
@carteblanchemunene 

What is this file for - can we delete it if not needed? 

cc @paulodhiambo  

image

**Review by paulodhiambo — APPROVED** [review]

**Comment by paulodhiambo** [comment]
Hello @hectorkambow I have removed the file.

**Comment by lalith1403** [comment]
Jumping on this one.

**Comment by lalith1403** [comment]
@paulodhiambo @carteblanchemunene There were several spelling mistakes. For example, CSV is spelt as CVS. Coding spelt as cording. Kindly look into such errors. 

@paulodhiambo Also look into the technical aspect of the article. There were several formatting errors with the code in the article. 

Overall, a nice article. 🚀

**Pull request merged**